manifest.json

```diff
@@ -540,14 +540,14 @@
         }
       },
       "waveSurferJs": {
-        "version": "1.0.46",
-        "downloadFormat": "files",
-        "url": "https://cdn.rawgit.com/katspaugh/wavesurfer.js/v1.0.46/dist/",
-        "rootDirPrefix": "waveSurfer-js-",
+        "version": "1.4.0",
+        "downloadFormat": "zip",
+        "url": "https://github.com/katspaugh/wavesurfer.js/archive/1.4.0.zip",
+        "rootDirPrefix": "wavesurfer.js-",
         "targetDirPrefix": "wave-surfer-js-",
-        "files": ["wavesurfer.min.js"],
+        "files": ["src/wavesurfer.js"],
```

Comment:
Should we use the `dist/` files? (have you tested this? I'm not sure whether `src/wavesurfer.js` needs other dependencies from `src/` dir.)

Also, can we use the `dist/wavesurfer.min.js` (at the end we are going to minify the file)

Reply:
Hi @DubeySandeep, I agree with your point we should use `dict/` I'll add and test with the CDN provided.